pro_1.py

```diff
@@ -0,0 +1,100 @@
+import json
+import Patterns
+import config_st
+import database_interaction
+import User
+import re 
+from re import findall
+database_file=open("database.txt","r+")
+dict_n=json.load(database_file)
+
+
+def logged_in():
+    print("succes")
+
+def log_in():
+    print("login page")
+def sign_up():
+   
+    print("=====signup page=====")
+
+    def enter_ur_name():
+     name=input("step1/4----enter ur full name ")
+     i=0
+     while(len(name)==0 or name==" ") :
+         name=input("ur input is empety enter your name again")
+        
+         i=i+1
```

Comment:
this will still stop after 1 trial, the second trial will take the input but wont break until it reaches the end of the cycle!

Reply:
yah ur right i fixed it thx